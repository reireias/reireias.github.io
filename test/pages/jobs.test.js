import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import CompositionApi from '@vue/composition-api'
import target from '@/pages/jobs'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)
localVue.use(CompositionApi)

describe('jobs', () => {
  it('should be vue instance', () => {
    const wrapper = shallowMount(target, {
      localVue,
      Vuetify,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
