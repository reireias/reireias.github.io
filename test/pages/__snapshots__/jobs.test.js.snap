// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jobs should be vue instance 1`] = `
"<v-container-stub tag=\\"div\\">
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <div class=\\"display-3\\">Job</div>
  </v-row-stub>
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <v-col-stub cols=\\"12\\" sm=\\"8\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" flat=\\"true\\">
        <v-card-title-stub class=\\"bottom-border-primary\\">
          職歴
        </v-card-title-stub>
        <v-card-text-stub>
          <div class=\\"title left-border-primary\\">
            富士ゼロックス株式会社（2013年〜2019年）
          </div>
          <ul>
            <li>
              機械翻訳サービス、中小企業向けBtoB向けクラウドストレージサービス、大企業向けクラウドストレージサービスなどを担当
            </li>
          </ul>
          <div class=\\"title left-border-primary\\">
            メドピア株式会社（2019年〜）
          </div>
          <ul>
            <li>
              SRE兼サーバーサイドエンジニアとして、サービスのコンテナ化やRuby
              on Railsでの実装を担当
            </li>
          </ul>
        </v-card-text-stub>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
  <v-row-stub tag=\\"div\\" justify=\\"center\\">
    <v-col-stub cols=\\"12\\" sm=\\"8\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" flat=\\"true\\">
        <v-card-title-stub class=\\"bottom-border-accent\\">
          学歴
        </v-card-title-stub>
        <v-card-text-stub>
          <div class=\\"title left-border-accent\\">
            筑波大学（2007年〜2011年）
          </div>
          <ul>
            <li>理工学群 工学システム学類</li>
            <li>学士（工学）を取得</li>
          </ul>
          <div class=\\"title left-border-info\\">
            筑波大学大学院（2011年〜2013年）
          </div>
          <ul>
            <li>システム情報工学研究科 リスク工学専攻</li>
            <li>修士（工学）を取得</li>
          </ul>
        </v-card-text-stub>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
</v-container-stub>"
`;
